evap/staff/tests.py

```diff
@@ -842,13 +846,14 @@ class SingleResultFormTests(TestCase):
 
     def test_single_result_form_saves_participant_and_voter_count(self):
         responsible = mommy.make(UserProfile)
+        course_type = mommy.make(CourseType)
         form_data = {
             "name_de": "qwertz",
             "name_en": "qwertz",
-            "type": "a type",
+            "type": course_type.id,
             "degrees": ["1"],
             "event_date": "02/1/2014",
-            "responsible": responsible.pk,
+            "responsible": responsible.id,
```

Comment:
i think `pk` is actually recommended, since it points to whatever field is used as primary key, which isn't necessarily called `id`